pages/game_page.py

```diff
@@ -1,3 +1,45 @@
 import streamlit as st
+import random 
 
-st.title("GAME PAGE")
\ No newline at end of file
+st.title("Build a Scientist")
+
+game_words = ['Genetic','Histology','Sequence','Mutation','Genome','Oncology']
+
+
+if game_words:
+    selected_word = random.choice(game_words)
+
+# Session state
+if 'selected_word' not in st.session_state:
+    st.session_state.selected_word = selected_word
+    
+if 'good_guesses' not in st.session_state:
+    st.session_state.good_guesses = []
+
+if 'bad_guesses' not in st.session_state:
+    st.session_state.bad_guesses = []
+
+# Display and update the game word as player enters new guesses 
+def game_word_generator(word):
+    st.write(word)
+    # Create a display string with underscores for each letter in the word
+    final_word = ' '.join([guess if guess in st.session_state.good_guesses else '_' for guess in word])
```

Comment:
To fix issue with first letter change to 
```suggestion
    final_word = ' '.join([guess if guess.lower() in st.session_state.good_guesses else '_' for guess in word])
```

Reply:
Ohhh makes way more sense this way

Reply:
JK i thought my changes fixed it but it didn't, checking why